### PR TITLE
Add changelog for update sig-storage-lib-external-provisioner to v8.0.1

### PR DESCRIPTION
**Pull request by sunnylovestiramisu**
**What type of PR is this?**

/kind documentation
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-3.2.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# Release notes for v3.2.1
```
Comment (msau42) [inline CHANGELOG/CHANGELOG-3.2.md:1]: We update the changelog in the release branch. There is already a 3.2.1

Comment (sunnylovestiramisu) [reply CHANGELOG/CHANGELOG-3.2.md:1]: It should be 3.2.2 then?

Comment (sunnylovestiramisu) [reply CHANGELOG/CHANGELOG-3.2.md:1]: The change log in https://github.com/kubernetes-csi/external-provisioner/blob/v3.2.0/CHANGELOG/CHANGELOG-3.2.md and https://github.com/kubernetes-csi/external-provisioner/blob/v3.2.1/CHANGELOG/CHANGELOG-3.2.md
 is identical?

Comment (msau42) [reply CHANGELOG/CHANGELOG-3.2.md:1]: hmm looks like we missed updating the changelog for v3.2.1 then.

**Pull request closed without merging**